gwt-uibinder-processor/pom.xml

```diff
@@ -51,6 +51,11 @@
       <groupId>com.google.guava</groupId>
       <artifactId>guava</artifactId>
     </dependency>
+    <dependency>
+      <groupId>com.google.auto</groupId>
+      <artifactId>auto-common</artifactId>
+      <version>${auto.common.version}</version>
+    </dependency>
```

Comment:
New dependency is unnecessary.   See other comments for replacement expressions.